modules/search/class.jetpack-search.php

```diff
@@ -862,11 +896,17 @@ public function add_post_type_aggregation_to_es_query_builder( array $aggregatio
 	 * @param Jetpack_WPES_Query_Builder $builder The builder instance that is creating the ES query
 	 */
 	public function add_date_histogram_aggregation_to_es_query_builder( array $aggregation, $label, Jetpack_WPES_Query_Builder $builder ) {
+		$args = array(
+			'interval' => $aggregation['interval'],
+			'field'    => ( ! empty( $aggregation['field'] ) && 'post_date_gmt' == $aggregation['field'] ) ? 'date_gmt' : 'date',
+		);
+
+		if ( isset( $aggregation['min_doc_count'] ) ) {
+			$args['min_doc_count'] = intval( $aggregation['min_doc_count'] );
+		}
+
 		$builder->add_aggs( $label, array(
-			'date_histogram' => array(
-				'interval' => $aggregation['interval'],
-				'field'    => ( ! empty( $aggregation['field'] ) && 'post_date_gmt' == $aggregation['field'] ) ? 'date_gmt' : 'date',
-			),
+			'date_histogram' => $args,
```

Comment:
I wonder if we should still have some validation / translation here? For example turning `post_date_gmt` into `date_gmt` automatically. If anyone is using `post_date_gmt` as their field here, it'll suddenly stop working.

Reply:
Oh nevermind me, I see that the logic was just moved up :)